### PR TITLE
Stitch story pages

### PR DESCRIPTION
**Pull request by Anushehzm**

### CODE REVIEW

File: EXP-Client/stories/Stories.js [hunk 1]
```diff
@@ -4,29 +4,25 @@ import { Provider, Appbar, FAB } from 'react-native-paper'
...
+  const [isPanelActive, setIsPanelActive] = React.useState(false);
   return (
     <View style={styles.content}>
       <Appbar.Header elevated>
         <Appbar.Content title="Stories" />
       </Appbar.Header>
       <StoryList stories={stories} setViewingStory={setViewingStory} />
-      <FAB style={styles.button} icon="plus" onPress={handleCreateStory} label="Create Today's Story" />
+      <FAB style={styles.button} icon="plus" onPress={() => setIsPanelActive(true)} label="Create Today's Story" />
+      <AddStoryMenu isPanelActive={isPanelActive} setIsPanelActive={setIsPanelActive} />
```
Comment (emmayjiang) [inline EXP-Client/stories/Stories.js:18]: These are still using our old buttons, could you update them to the Material ones?

image

File: EXP-Client/stories/NewStory.js [hunk 2]
```diff
@@ -6,19 +6,22 @@ import {
...
 registerTranslation('en', en);
 
-export default function AddStoryMenu({ isPanelActive, setIsPanelActive }) {
+export default function AddStoryMenu({isPanelActive, setIsPanelActive }) {
+  const navigation = useNavigation();
   const [name, setName] = React.useState('');
-  const [dateCreated, setDeadline] = React.useState();
+  const [dateCreated, setDeadline] = React.useState('23rd April 2023'); // Dummy date for testing purposes
 
   const handleAddTodo = () => {
```
Comment (emmayjiang) [inline EXP-Client/stories/NewStory.js]: Don't know if this is within the escape of this PR, but I'm getting errors trying to click this.
image

**Review by emmayjiang — CHANGES_REQUESTED** [review]
Running into errors + a nitpick

File: EXP-Client/stories/Stories.js (hunk 1 above)
Comment (Anushehzm) [reply EXP-Client/stories/Stories.js:18]: Done

**Review by emmayjiang — APPROVED** [review]
Thanks for making the changes!

**Pull request merged**